src/pre_defined_classes/mathematical_program/medium/q4/BasketballGamesTest.java

```diff
@@ -0,0 +1,31 @@
+package pre_defined_classes.mathematical_program.medium.q4;
+
+import global.BaseRandomTest;
+import global.variables.Clause;
+import global.variables.clauses.IntegerLiteral;
+import global.variables.clauses.NewLine;
+import global.variables.clauses.StringLiteral;
+
+public class BasketballGamesTest extends BaseRandomTest {
+
+    public Clause[] testSentence() {
+        return new Clause[]{
+                new StringLiteral("Our team scored "),
+                new IntegerLiteral(127),
```

Comment:
We can also take advantage of ranges to make this less hardcode-y

Reply:
I am unsure what you mean by this